.gitignore

```diff
@@ -21,5 +21,5 @@ build/
 # temporary build files create by air
 tmp/
 
-# dotenv variables
+# env file
 .env
```

Comment:
Why should this be gitignored?

If it's okay, let's checkin the default .env file, it will be easier for others to run the code my making minimal edits.

Reply:
It shouldn't be a problem right now. But since we also store secrets as env variables, I just added it to gitignore. Also, I think generally most projects keep `.env` in `.gitignore`.

Reply:
I think it would be better to store these values as constants inside a go file instead of committing the `.env` file to git.